packages/ui5-task-zipper/lib/zipper.js

```diff
@@ -21,6 +21,24 @@ const determineProjectName = (collection) => {
 	return projectName || collection._project?._name /* UI5 tooling 3.x */ || collection._project?.metadata?.name; /* UI5 tooling 2.x */
 };
 
+/**
+  * Turn absolute data source paths in the manifest.json into relative paths
+  *
+  * @param {Buffer} buffer Buffer of the manifest.json file
+  * @param {object} zip ZipFile instance
+  * returns {void}
+*/
+const absoluteToRelativePaths = (buffer, zip) => {
+	const manifest = JSON.parse(buffer.toString("utf-8"));
+	if (!manifest["sap.app"]["dataSources"]) return;
+	for (const [dataSourceName, dataSource] of Object.entries(manifest["sap.app"]["dataSources"])) {
+		if (dataSource.uri?.substring(0, 1) === "/") {
```

Comment:
usage of [`charAt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt) would be a little bit shorter

Reply:
Yep